src/container.cpp

```diff
@@ -736,27 +736,27 @@ void Container::startDecaying()
 {
 	Item::startDecaying();
 
-	for (Item* item : itemlist) {
+	for (auto item : itemList) {
```

Comment:
Could this be `const auto&`
```suggestion
	for (const auto& item : itemList) {
```